examples/sdk-app-example/build.gradle

```diff
@@ -38,14 +66,9 @@ android {
     externalNativeBuild.cmake.path "CMakeLists.txt"
 }
 
-project.afterEvaluate {
-    project.android.lintOptions.abortOnError = false
-}
-
 dependencies {
-    implementation "com.bugsnag:bugsnag-android:+"
-
-    implementation "androidx.appcompat:appcompat:1.0.2"
+    implementation "com.bugsnag:bugsnag-android:9.9.9"
```

Comment:
Will this get set to a real version when the spec refactor has properly landed?

Reply:
Yep - the intention is to update this to a real version when we release the spec refactor.